src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -248,6 +248,11 @@ public MappingProfiles()
 
     private static string GetDataSetUploadStatus(DataSetScreenerResponse screenerResult)
     {
+        if (screenerResult is null)
+        {
+            return DataSetUploadStatus.SCREENER_ERROR.ToString();
```

Comment:
I think nowadays we get warnings from Rider to say to use `nameof(DataSetUploadStatus.SCREENER_ERROR)` if dealing with enum strings.